src/main/java/io/appium/java_client/remote/AppiumCommandExecutor.java

```diff
@@ -191,8 +191,8 @@ private Response createSession(Command command) throws IOException {
         return response;
     }
 
-    public void refreshAdditionalCommands() {
```

Comment:
renaming it would be a breaking change. Can we keep the method and just fix its implementation instead?